src/pages/fr/guides/deploy/github.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Déployez votre site Astro sur les Github Pages
+description: Comment déployer votre site Astro sur le Web à l'aide des Github Pages.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Vous pouvez utiliser les [GitHub Pages](https://pages.github.com/) pour héberger un site Astro directement depuis un dépôt sur [GitHub.com](https://github.com/).
+
+## Comment déployer
+
+Vous pouvez déployer un site Astro sur les GitHub Pages en utilisant une [GitHub Actions](https://github.com/features/actions) afin d'automatiser le build et le déploiement de votre site. Pour faire ça, votre code source doit être hébergé sur GitHub.
+
+Astro maintient l'action officielle `withastro/action` pour déployer vos projets avec très peu de configuration. Suivez les instructions ci-dessous pour déployer votre site Astro sur les Github Pages, et lisez [le package README](https://github.com/withastro/action) si vous souhaitez plus d'informations.
+
+1. Définissez les options [`site`](/fr/reference/configuration-reference/#site), et si besoin, [`base`](/fr/reference/configuration-reference/#base) dans `astro.config.mjs`.
+    - `site` devrait être quelque chose comme `https://<YOUR_USERNAME>.github.io`
+    - `base` doit être le nom de votre dépôt commençant par un slash, par exemple `/my-repo`.
+    
+    :::note
+    Si votre dépôt s'appelle `<YOUR_USERNAME>.github.io`, vous ne devez pas inclure `base`.
+    :::
+
+2. Créez un nouveau fichier dans votre projet à `.github/workflows/deploy.yml` et collez-y le YAML ci-dessous.
+
+    ```yaml
+    name: Github Pages Astro CI
+
+    on:
+      # Trigger the workflow every time you push to the `main` branch
+      # Using a different branch name? Replace `main` with your branch’s name
+      push:
+        branches: [ main ]
+      # Allows you to run this workflow manually from the Actions tab on GitHub.
+      workflow_dispatch:
+      
+    # Allow this job to clone the repo and create a page deployment
```

Comment:
What do you think about translating these comments so the reader knows what it's happening without needing to know English? (you don't need to translate the action's steps)

Reply:
Yeah, good idea ! I'll do that !

Reply:
Done. https://github.com/withastro/docs/pull/1349/commits/64de601e30f17ea53b7017eecb76ded21a59941c